src/integrators.py

```diff
@@ -13,24 +13,24 @@ class Integrator:
     def __init__(
         self,
         # bounds: Union[List[Tuple[float, float]], np.ndarray],
-        map,
+        map=None,
```

Comment:
How about the setting "map: Map = None"?

Reply:
How about map: Union[List[List[float]], List[Tuple[float, float]], np.ndarray, Map] = None? If map is not Map rather bounds, we set map to Affine(Map).